skins/laika/src/components/Question.vue

```diff
@@ -1,31 +1,30 @@
 <template>
-    <div class="question space-above">
-        <div v-bind:class="[ isOpen ? 'border-secondary' : 'border-primary' ]">
-            <div class="inline-icon" @click="toggle()"
-                 :data-content-target="!isOpen ? '/page/Häufige Fragen' : ''"
-                 :data-track-content="!isOpen"
-                 :data-content-name="!isOpen ? 'Expand' : ''"
-                 :data-content-piece="!isOpen ? content.question : ''">
```

Comment:
We use these data attributes to track how often the questions are opened.
Unless this is not a requirement anymore they should be put back in the `<div>`